msdk-featdet/msdk-featdet-msmsdetection/src/main/java/io/github/msdk/featdet/msmsdetection/MsMsDetectionMethod.java

```diff
@@ -116,8 +116,7 @@ public MsMsDetectionMethod(@Nonnull RawDataFile rawDataFile,
             // Isolation m/z for MS/MS scan (Precursor ion)
             // We assume a single isolation since only MS2 scans are accepted
```

Comment:
I think scans with no isolations should be safely ignored. Now the code will throw an exception.

Also, I think the method should take the last isolation in the list, not the first one. Imagine MS3 fragmentation - you want to get the precursor value of the last fragmentation.